src/mlagility/helpers/labels.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Dict, List
+
+
+def load_from_file(file_path: str) -> Dict[str, List[str]]:
+    """
+    This function extracts labels from a Python file.
+    Labels must be in the first line of a Python file and start with "# labels: "
+    Each label must have the format "key::value1,value2,..."
+
+    Example:
+        "# labels: author:google test_group:daily,monthly"
+    """
+    # Labels must be in the first line of the python file
```

Comment:
Update #24 with any rules like this that need to be documented in the `models/readme.md`

Reply:
updated!